utilities/validations.py

```diff
@@ -0,0 +1,27 @@
+from api.location.models import CountryType, TimeZoneType
```

Comment:
Since we have various validations in the codebase other than for a country field and timezone, the file name would be misleading if it only contains these two methods unless you are planning to move all the existing validation methods into this new file.

Reply:
Yes I concur. Let me move all of them into this file.

Reply:
Or you could just rename it to what it specifically does

Reply:
Yes I get you. The initial thought was create this file for validations of data drawn from models. I'll move the validations into this file to prevent the confusion.